_posts/2023-04-14-kafka-transactions.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: post
+title: "Kafka transactions (exactly-once semantics)"
+date: 2023-04-14
+author: federico_valeri
+---
+
```

Comment:
You've got some good context here, but what's missing in the first two or three paragraphs is telling the reader what they're going to learn from the rest of this post. What is the point you're trying to inform them about/why should they care?

Reply:
I'll try to improve that. In addition to learn how it works, I thinks drawbacks and how the txn metadata is stored, which helps in troubleshooting.

Thanks.